src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -248,8 +248,9 @@
     private ResourceLocation texture;
     private int              colonyId;
     private int citizenId = 0;
-    private int          level;
-    private int          textureId;
+    private int level;
+    private int textureId;
+    private float volume = 0.2F;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.2F to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)